pkg/util/node/node.go

```diff
@@ -106,17 +107,31 @@ func GetNodeHostIP(node *v1.Node) (net.IP, error) {
 }
 
 // GetNodeIP returns the ip of node with the provided hostname
+// If required, wait for the node to be defined.
 func GetNodeIP(client clientset.Interface, hostname string) net.IP {
 	var nodeIP net.IP
-	node, err := client.CoreV1().Nodes().Get(hostname, metav1.GetOptions{})
-	if err != nil {
```

Comment:
I think it can try to get node at first.
If error then use ``ExponentialBackoff``.

Reply:
Can't we avoid adding same piece of codes present inside `ExponentialBackoff`'s `func` for getting node and nodeIP here. I don't see any advantage of keeping them outside. `ExponentialBackoff` will execute the `func` without any delay for the 1st time and if it passes, it will break the loop the immediately.